backend/onedrive/quickxorhash/xor.go

```diff
@@ -0,0 +1,20 @@
+//go:build !1.20 || !1.21 || !1.22
```

Comment:
This build constraint should say just

    //go:build !go1.20

I think.

That means go version < 1.20

Reply:
Thanks for suggestions! I don't use go's build constraints very often so I'm not familiar with them. Applied in last amend-commit. thanks again 🍻